contracts/AxelarAuthMultisig.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { ECDSA } from './ECDSA.sol';
+import { IAxelarAuthMultisig } from './interfaces/IAxelarAuthMultisig.sol';
+
+contract AxelarAuthMultisig is IAxelarAuthMultisig {
+    error NotGateway();
+    error InvalidOperators();
+    error InvalidThreshold();
+    error SameOperators();
+    error MalformedSigners();
+
+    address public gateway;
+    uint256 public currentEpoch;
+    mapping(uint256 => bytes32) public hashForEpoch;
+    mapping(bytes32 => uint256) public epochForHash;
+
+    uint8 internal constant OLD_KEY_RETENTION = 16;
+
+    constructor(address gatewayAddress, bytes memory operatorsData) {
```

Comment:
```suggestion
    constructor(bytes memory operatorsData) {
```
I'd rather not pass in the gateway's address to keep the deployment identical for chains already having gateway deployed and not.

Reply:
Makes sense